src/Stryker.Core/Stryker.Core.UnitTest/MutantFilters/IgnoredMethodMutantFilterTests.cs

```diff
@@ -74,19 +77,19 @@ private void TestMethod()
         }
 
         [Theory]
-        [InlineData("Where", true)]
-        [InlineData("Where*", true)]
-        [InlineData("*Where", true)]
-        [InlineData("*Where*", true)]
-        [InlineData("*ere", true)]
-        [InlineData("Wh*", true)]
-        [InlineData("W*e", true)]
-        [InlineData("*", true)]
-        [InlineData("ToList", false)]
-        [InlineData("*List", false)]
-        [InlineData("To*", false)]
-        [InlineData("T*ist", false)]
-        [InlineData("", false)]
+        [InlineData("^Where$", true)]
+        [InlineData("^Where.*$", true)]
```

Comment:
In general, I would suggest splitting this test into 2 theories, for skipping and not skipping mutants, as logic in tests is sometimes [frowned upon](https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices#avoid-logic-in-tests) :)

But if the practice is ubiquitous here, maybe not necessary to address in this PR.